zerver/openapi/zulip.yaml

```diff
@@ -16836,6 +16854,25 @@ paths:
                                     a channel, we will send an empty array. API authors
                                     should use other data to determine whether users like
                                     guest users are forbidden to know the subscribers.
+                                partial_subscribers:
+                                  type: array
+                                  items:
+                                    type: integer
+                                  description: |
+                                    A list of user IDs of a subset of users who are subscribed
+                                    to a given channel, specifically bots and recently active users.
+                                    Included only if `include_subscribers` is `partial` and number of subscribers
+                                    in the channel is greater than `MIN_PARTIAL_SUBSCRIBERS_CHANNEL_SIZE`
+                                    [server setting][server-settings] which defaults to 1000.
+
+                                    If a user is not allowed to know the subscribers for
+                                    a channel, we will send an empty array. API authors
+                                    should use other data to determine whether users like
+                                    guest users are forbidden to know the subscribers.
+
+                                    [server-settings]: https://zulip.readthedocs.io/en/stable/production/settings.html
```

Comment:
This description should mainly reference the new `include_subscribers` description. Ideally, with a link. But text could be something like:

> If `include_subscribers="partial"` was requested, the server may, at its discretion, send a `partial_subscribers` list rather than a `subscribers` list for channels with large numbers of subscribers. The `partial_subscribers` is an arbitrary subset of the channel's subscribers that is guaranteed to include all bot user subscribers as well as all users who've been active in the last month, but otherwise can be chosen arbitrarily be the server.